app/services/api-call.service.ts

```diff
@@ -58,12 +67,18 @@ export class ApiCall {
 
     let req: Request = new Request(options);
     let authorizationData = this.localStorageWrapper.getObject(this.storageAuthorizationData);
-    if (authorizationData) {
+    if (authorizationData != null) {
```

Comment:
use !! instead of != null for consistency with other places. And it looks cleaner.